mantle/kola/tests/misc/raid.go

```diff
@@ -144,10 +144,13 @@ type lsblkOutput struct {
 }
 
 type blockdevice struct {
-	Name       string        `json:"name"`
-	Type       string        `json:"type"`
-	Mountpoint *string       `json:"mountpoint"`
-	Children   []blockdevice `json:"children"`
+	Name       string  `json:"name"`
+	Type       string  `json:"type"`
+	Mountpoint *string `json:"mountpoint"`
+	// remove this field once upstream addresses
+	// backward compatibility issue for `Mountpoint`
```

Comment:
@jlebon Just saw this comment from @karelzak regarding `Mountpoint` https://github.com/karelzak/util-linux/issues/1272#issuecomment-814776311

Reply:
OK right, so actually an alternative fix here would've been `lsblk --json -o +mountpoint` to restore the field in the output. But `mountpoints` is more correct anyway so I'd just leave the current behaviour you have here.

But maybe let's update the comment to something like:

```
// new lsblk outputs `mountpoints` instead of `mountpoint`; we handle both
```

since there's nothing for upstream to actually address.